frontend/tests/e2e/member/filter/active-on.js

```diff
@@ -0,0 +1,63 @@
+export default () => {
+  before(() => {
+    cy.wait(1000);
+    cy.get('.filter-dropdown button').click();
+    cy.get('#filterList li').contains('Active On').click();
+  });
+
+  beforeEach(() => {
+    cy.scrollTo(0, 0);
+    cy.server();
+    cy.route('POST', '/api/tenant/*/member/query').as('apiMemberQuery');
+  });
+
+  after(() => {
+    cy.scrollTo(0, 0);
+    cy.get('.filter-list .filter-list-item:first-child button:last-child').click({ force: true });
+  });
+
+  it('has apply button disabled if no platform selected', () => {
+    cy.get('.filter-type-select + div button.btn--primary').should('be.disabled');
```

Comment:
One thing I applied in the past when tests with react-testing-library was never to use classes to get elements, specially classes with styles such as `btn--primary`. Classes are much more likely to changes, specially style ones. As soon as you change a style class tests will fail. I used to use `data-qa` attribute, just searched in cypress documentation and this is what they recommend
https://docs.cypress.io/guides/references/best-practices#Selecting-Elements

Let me know what you think!